nix/sources.json

```diff
@@ -5,10 +5,10 @@
         "homepage": null,
         "owner": "input-output-hk",
         "repo": "cardano-wallet",
-        "rev": "5c1240cc668993fcb7135f244f2b1cc9c96a3238",
-        "sha256": "0fwgigcl11jwr34pbv92099z67j18wr2132a1zbr19llkg220mlr",
+        "rev": "efd056bb720c137f44e88cfbf01abb28cda6fed4",
```

Comment:
This isn't a release commit, it's just the latest master...

Reply:
That's fine.